### DISCUSSION TITLE
how to avoid Failed to initialize Python.Runtime.dll

### DISCUSSION THREAD

**Discussion by danielhrisca** [pythonnet/pythonnet#1735] [Q&A]
Hello all,

I get this error on a PC that probably has the locale set to German.
The error occurs on this line https://github.com/pythonnet/pythonnet/blob/eec30c062f93c8555bcff9ec3bb14ccc832f36df/pythonnet/__init__.py#L43 and I suspect the intent here is to see if the default encoding is `utf-8`. 

How can I get around this problem?

**Comment by filmor** [comment]
No, that's not the intent. This is the same as `b""`, it's just empty for now because we don't have any useful content for this parameter right now and the `.encode("utf8")` is there to remind us of the correct encoding :)

**Reply by danielhrisca** [reply]
Understood, any idea what else could be wrong? Running `reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\NET Framework Setup\NDP"` gives this output

![image](https://user-images.githubusercontent.com/20952040/158856183-a000ad08-4b93-4c0e-b7cd-69bb7b2bb94b.png)

**Reply by danielhrisca** [reply]
@filmor could it be related to the Windows 10 Evergreen version?
I bundle a python application using PyInstaller on a PC that runs Windows 10 Evergreen version 20H2. The application runs OK on other PCs that have 20H2 (as far as I've tested), but the problem occurs on a PC that has Windows 10 Evergreen 21H1.

**Reply by danielhrisca** [reply]
I was using a wheel that I had created locally from the latest master branch code, and this was not working on the mentioned PC.
I've now used the PyPI 3.0.0.a2 wheel and it is working correctly.

**Reply by TTSPhilipp** [reply]
Hello @danielhrisca,

I think I have a similar problem. Have you been able to find out if the problem is caused by the Windows version?
For me the error occurs with the Windows version 20H2.

**Reply by danielhrisca** [reply]
The latest 3.0 release works for me now

**Reply by TTSPhilipp** [reply]
Oh, that's great, of course. Thanks for the quick response.
